sudoers/init.sls

```diff
@@ -2,15 +2,15 @@
 
 sudo:
   pkg.installed:
-    - name: {{ sudoers.pkg }}
+    - name: {{ sudoers.lookup.pkg }}
```

Comment:
Using the same sudoers variable for init and template requires specifying the lookup portion.